app/controllers/users_controller.rb

```diff
@@ -1,17 +1,24 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: %i(show new create)
+  before_action :load_user, only: %i(index new create)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: :destroy
+
+  def index
+    @users = User.all.paginate(page: params[:page])
```

Comment:
```suggestion
    @users = User.page(params[:page]).per 5
``` and don't forget to add 5 to settings.yml file and call here